store/interfaces.json

```diff
@@ -15,5 +13,44 @@
       "product-specifications"
     ],
     "component": "index"
+  },
+  "product-assembly-options": {
+    "component": "ProductAssemblyOptions",
+    "composition": "children",
+    "allowed": [
+      "product-assembly-options.image",
+      "product-assembly-options.quantity-selector",
+      "product-assembly-options.name",
+      "product-assembly-options.price",
+      "product-assembly-options.customize",
+      "product-assembly-options.children-description"
```

Comment:
I think those interfaces are not extensions of product-assembly-options but a new interface.